README.md

```diff
@@ -9,9 +9,9 @@
 
 ## Proven React Native boilerplate
 
-The culmination of over seven years of constant React Native development, Ignite is the most popular React Native app boilerplate for both Expo and bare React Native.
+Having been consistently developed and maintained since 2016, Ignite is the oldest active and most popular third-party React Native / Expo app boilerplate.
```

Comment:
```suggestion
Developed and maintained consistently since 2016, Ignite is the oldest active and most popular third-party React Native / Expo app boilerplate.
```

Reply:
My 10th grade English teacher made forever aware of avoiding the passive voice :)